app/models/wpcc/your_details_form.rb

```diff
@@ -1,15 +1,15 @@
 module Wpcc
   class YourDetailsForm
-    extend Wpcc::SalaryThreshold
     include ActiveModel::Model
 
     attr_accessor :age, :gender, :salary
     attr_accessor :salary_frequency, :contribution_preference
 
     AGE = { minimum: 16, maximum: 74 }.freeze
     GENDERS = %w[male female].freeze
-    SALARY_FREQUENCIES = load_config(file: 'salary_threshold').keys.map(&:to_s)
     CONTRIBUTIONS = %w[full minimum].freeze
```

Comment:
Is it worth using the configLoader for the other constants e.g. AGE, GENDERS, CONTRIBUTIONS?

Reply:
Gender and age I don't think it is necessary to use a yaml because we are using only for validation and nothing extra.